### ISSUE TITLE
Settings for a default option name for processing options

### ISSUE THREAD

**Issue by bonafernando** [imgproxy/imgproxy#153]
Is it possible to have a setting to allow us to hide the `option_name` of a certain `processing_options`?
Example: set `pr` as the `DEFAULT_PROCESSING_OPTIONS` or `DEFAULT_OPTION_NAME`, and then all urls like this `/100x100/` would internally be interpreted as `/pr:100x100/`.

Thank you.

**Comment by carlosbrando** [comment]
Here we only allow presets in the URLs, so the `pr:` is entirely unnecessary for us.

**Comment by bonafernando** [comment]
Does something like this would make sense to you? @DarthSim 
![image](https://user-images.githubusercontent.com/2565620/56815491-e3638f00-6817-11e9-9e7d-1c4a2ab65df6.png)
```
		default_option_name := ""
		strEnvConfig(default_option_name, DEFAULT_OPTION_NAME)
		if len(default_option_name) > 0 && len(args) == 0 {
			return applyProcessingOption(po, DEFAULT_OPTION_NAME, []string{name})
		}
```
Below this line: https://github.com/imgproxy/imgproxy/blob/master/processing_options.go#L693

**Comment by bonafernando** [comment]
Rethinking here... I'm afraid It would never fall on this path if `args` is 0. It requires changes somewhere else.

**Comment by DarthSim** [comment]
Implemented in #156
